index.js

```diff
@@ -21,13 +25,22 @@ const userSchema = new mongoose.Schema({
     email: String,
     password: String,
     id: Number,
+    role:String,

```

Comment:
what is this role, is it any new column in mongodb. If yes, send that file as well.

Reply:
I will  send user.json file ,upload this in your mongodb